config/testgrids/kubernetes/sig-storage/config.yaml

```diff
@@ -0,0 +1,125 @@
+      alert_options:
+        alert_mail_to_addresses: kubernetes-sig-storage-test-failures@googlegroups.com
+    - name: gke-alpha-enabled-default
+      test_group_name: ci-kubernetes-e2e-gci-gke-alpha-enabled-default
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage gci gke alpha enabled default features e2e tests for master branch
+    - name: gce-multizone
+      test_group_name: ci-kubernetes-e2e-gce-multizone
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage gce multizone e2e tests for master branch
+      alert_options:
+        alert_mail_to_addresses: kubernetes-sig-storage-test-failures@googlegroups.com
+    - name: gke-multizone
+      test_group_name: ci-kubernetes-e2e-gci-gke-multizone
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage gci gke multizone e2e tests for master branch
+    - name: gke-regional
+      test_group_name: ci-kubernetes-e2e-gke-regional
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage gke regional e2e tests for master branch
+    - name: gke-regional-serial
+      test_group_name: ci-kubernetes-e2e-gke-regional-serial
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage gke regional serial e2e tests for master branch
+    - name: gke-regional-slow
+      test_group_name: ci-kubernetes-e2e-gke-regional-slow
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage gke regional slow e2e tests for master branch
+    - name: gce-iscsi
+      test_group_name: ci-kubernetes-e2e-gce-iscsi
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage iSCSI e2e tests for master branch
+      alert_options:
+        alert_mail_to_addresses: kubernetes-sig-storage-test-failures@googlegroups.com
+    - name: gce-iscsi-serial
+      test_group_name: ci-kubernetes-e2e-gce-iscsi-serial
+      base_options: include-filter-by-regex=Volume%7Cstorage
+      description: storage iSCSI serial e2e tests for master branch
+      alert_options:
+        alert_mail_to_addresses: kubernetes-sig-storage-test-failures@googlegroups.com
+
+- name: sig-storage-local-static-provisioner
+  dashboard_tab:
+    - name: master-gce-lastest
+      test_group_name: ci-sig-storage-local-static-provisioner-master-gce-latest
+      base_options: include-filter-by-regex=sig-storage
+      description: E2E tests against latest kubernetes on GCE
+      alert_options:
+        alert_mail_to_addresses: kubernetes-sig-storage-test-failures@googlegroups.com
+    - name: master-gke-default
+      test_group_name: ci-sig-storage-local-static-provisioner-master-gke-default
+      base_options: include-filter-by-regex=sig-storage
+      description: E2E tests against default kubernetes on GKE
+
+- name: sig-storage-csi
```

Comment:
are there a bunch of jobs missing here? https://k8s-testgrid.appspot.com/sig-storage-csi

Reply:
Prow Jobs are usually configured using annotations. As long as a dashboard is declared here, jobs can simply declare that they want their tests to show up on that dashboard.

[Example for sig-storage-csi](https://github.com/kubernetes/test-infra/blob/f0114293799b2f2b5b29fbf19f44d3e10882234d/config/jobs/kubernetes-csi/csi-driver-host-path/csi-driver-host-path-config.yaml#L339)
[Doc](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md)